x/poolmanager/taker_fee_test.go

```diff
@@ -19,7 +19,7 @@ func (s *KeeperTestSuite) TestChargeTakerFee() {
 
 	var (
 		defaultTakerFee = osmomath.MustNewDecFromStr("0.01")
-		defaultAmount   = sdk.NewInt(100)
+		defaultAmount   = sdk.NewInt(10000000)
```

Comment:
This was increased because with a taker fee as small as 0.01 and an input of 100, the split between taker fee and community pool is too small to actually test. In other words, the taker fee is 1, and splitting this between the two does not work (in practice/mainnet what happens is, the first category of staking rounds it down to 0, and then the remainder (1) is all sent to the second category)